learnify/web/src/App.js

```diff
@@ -2,8 +2,8 @@ import { BrowserRouter, Routes, Route} from 'react-router-dom';
 import './App.css';
 import EmailVerificationPage from './pages/EmailVerificationPage';
 import Header from './pages/Header';
+import LoginForm from './pages/LoginForm';
```

Comment:
import LoginForm from './pages/LoginForm' does this working since it gives me error because the file name is LogInForm.js in the repository